node/get_status_node.go

```diff
@@ -312,7 +312,7 @@ func (n *StatusNode) setupRPCClient() (err error) {
 	if err != nil {
 		return
 	}
-	n.rpcClient, err = rpc.NewClient(gethNodeClient, n.config.NetworkID, n.config.UpstreamConfig, n.config.Networks, n.appDB)
+	n.rpcClient, err = rpc.NewClient(gethNodeClient, n.config.NetworkID, n.config.UpstreamConfig, n.config.Networks, n.config.NimbusProxyConfig.Enabled, n.appDB)
```

Comment:
I wonder if at this point if it wouldn't be a good idea to create a config object instead of adding more parameters to the function?